js/foam/u2/md/CitationView.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2.md',
+  name: 'CitationView',
+  extends: 'foam.u2.md.DetailView',
+
+  properties: [
+    {
+      name: 'model',
+      defaultValueFn: function() {
+        return this.data.model_;
+      }
+    },
+  ],
+
+  methods: [
+    function pickNameProperty() {
```

Comment:
This looks expensive.  Can we cache this and then invalidate when data is updated?